src/index.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import Sidenav from './index'
+
+let props
+
+const dummyElement = (elValue) => React.createElement('li', { id: 'li1' }, elValue)
+
+describe('Index', () => {
+  beforeEach(() => {
+    props = {
+      sidenav: dummyElement('one'),
+      children: dummyElement('two'),
+      onSetOpen: jest.fn(),
+      open: false
+    }
+  })
+
+  it('renders without crashing', () => {
+    const wrapper = shallow(<div><Sidenav sidenav={props.sidenav} children={props.children} onSetOpen={props.onSetOpen} open={props.open} /></div>)
```

Comment:
```suggestion
     const wrapper = shallow(<div><Sidenav sidenav={props.sidenav} onSetOpen={props.onSetOpen} open={props.open}>{props.children}</Sidenav></div>)
```
It might be easier to understand if we use child element instead of `children` like this.